pkgs/development/python-modules/poetry/default.nix

```diff
@@ -0,0 +1,76 @@
+{ lib, buildPythonPackage, fetchPypi, callPackage
+, isPy27, isPy34
+, cleo
+, requests
+, cachy
+, requests-toolbelt
+, pyrsistent
+, pyparsing
+, cachecontrol
+, pkginfo
+, html5lib
+, shellingham
+, tomlkit
+, typing
+, pathlib2
+, virtualenv
+, functools32
+, pytest
+}:
+
+let
+  cleo6 = cleo.overrideAttrs (oldAttrs: rec {
+    version = "0.6.8";
```

Comment:
We usually can't have a Python package depend on a version that's not the default because this can lead to conflicts. Was there a reason we allowed this back then?
And isn't poetry only ever used as an application, i.e. never imported? In that case, we should move it out of `pythonPackages`.

Reply:
> We usually can't have a Python package depend on a version that's not the default because this can lead to conflicts. Was there a reason we allowed this back then?
And isn't poetry only ever used as an application, i.e. never imported?

Yeah it is weird that I forgot about that, especially since we're always mentioning this :smile: 
I can move this out of `python-modules`.

Reply:
If you're it's never imported, feel free to go ahead. Thank you in advance!